hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Hangman.Game do
+  @moduledoc """
+  This module contains the implementation of the Hangman API
+  """
+  import Hangman.Util
+
+  defstruct turns_left: 7, game_state: :initializing,
+            word: "", used: [], last_guess: '', letters: []
+
+  @doc """
+  Returns the initial state for a game
+  """
+  def new_game do
+    word = Dictionary.random_word()
+    letters = for _ <- String.codepoints(word), do: "-"
+    %Hangman.Game{}
```

Comment:
## -1

More idiomatic to say

~~~ elixir
%Hangman.Game{
    word:    word,
    letters: letters
}
~~~